package.json

```diff
@@ -96,7 +96,7 @@
       "gofmt -s -w",
       "yarn golint"
     ],
-    "*.{js,css,json,ts}": [
+    "*.{js,css,json,ts,tsx}": [
```

Comment:
Note: this is just for the commit hook stage. Not quite sure if within a commit this really catches everything. Well. Let's leave this here.